hyperspy/io_plugins/tiff.py

```diff
@@ -570,7 +577,7 @@ def get_mapping_Zeiss(self):
             ("Acquisition_instrument.SEM.Stage.rotation", self._parse_tuple_Zeiss),
             'CZ_SEM.ap_stage_at_t':
             ("Acquisition_instrument.SEM.Stage.tilt", self._parse_tuple_Zeiss),
-            'CZ_SEM.ap_free_wd':
```

Comment:
ap_free_wd parameter is absent in older versions of zeiss tiff, ap_wd is more correct.